base_agent/loco_mc_agent.py

```diff
@@ -138,6 +138,11 @@ def send_text_command_to_agent(sid, command):
             }
             sio.emit("setChatResponse", payload)
 
+        @sio.on("receiveHandshake")
+        def receive_handshake(sid, handshake):
```

Comment:
The `handshake` argument is never used in this function. It's directly assigned a value and sent back to dashboard. 
Generally if one argument is never used, then do not pass it. In this case, you can verify the handshake message first (e.g. check if it matches a given pattern like 'ping'), log it, then reply back with another message (e.g. 'pong' or received).